### ISSUE TITLE
Large performance gap between the validation and test sets

### ISSUE THREAD

**Issue by zhanggang001** [PJLab-ADG/DetZero#3]
Thanks for releasing the codes of this excellent work. 

I've observed that the L2 mAPH on the validation set is 81.7, whereas the L2 mAP on the test set is 85.15. According to my experience, If we use the same model for both Waymo validation and test sets, we should expect only a small gap in performance. Could you kindly explain why the performance on the test set is more than 3 points higher than that on the validation set? What are the differences between these two models? Thank you in advance.

**Comment by superkoma** [comment]
Thanks for your interest in our work. There are a few differences between these two results.
- Based on my experience with the Waymo dataset, if your model is not overfitting, it would have a 1.2~2 points improvement on test set, compared to val set.
- The amounts of PED especially CYC are quite small, which leads to an unstable training process of refining the model. Hence, we add val set data in our training set to seek more improvements for leaderboard results (just for the leaderboard), which are very common techniques used in KITTI, nuscenes, waymo.
- More model ensembles, as you can see, our leaderboard result was submitted very closed to ICCV deadline, there is little time for us to conduct the ablation experiments related results on val set once more.

There are no other technical differences, all the models' training follows the same procedure. The leaderboard performance would be further improved if you use more models or TTA results for ensembling.

**Comment by zhanggang001** [comment]
> Thanks for your interest in our work. There are a few differences between these two results.
> 
> * Based on my experience with the Waymo dataset, if your model is not overfitting, it would have a 1.2~2 points improvement on test set, compared to val set.
> * The amounts of PED especially CYC are quite small, which leads to an unstable training process of refining the model. Hence, we add val set data in our training set to seek more improvements for leaderboard results (just for the leaderboard), which are very common techniques used in KITTI, nuscenes, waymo.
> * More model ensembles, as you can see, our leaderboard result was submitted very closed to ICCV deadline, there is little time for us to conduct the ablation experiments related results on val set once more.
> 
> There are no other technical differences, all the models' training follows the same procedure. The leaderboard performance would be further improved if you use more models or TTA results for ensembling.

Thank you very much for your gracious response.
